examples/http-send-different-status-codes-with-payload/http_send_different_status_codes_with_payload.md

```diff
@@ -1,6 +1,8 @@
 # REST service - Send different status codes with payload
 
-In addition to returning `StatusCodeResponse` constants, the resource method can return `StatusCodeResponse` record that contains response headers and payload. Defining a custom record type including the `StatusCodeResponse` can produce a better representation of the response in the OpenAPI specification. In addition to that, using typed records provides compiler validations and better tooling support.
+The resource method can return a custom status code response record with a body and headers.
+This custom record type can be created by including a subtype of the `http:StatusCodeResponse` record. This record type has a `body` field, which represents the response payload, and a `headers` field, which represents a `map` of the response headers. In addition, the `body` field type can be overwritten by a custom record type.
```

Comment:
Breaking the description into multiple sentences will add more space in web layout. Shall we concat and keep a single sentence?

Or is this intentional?

Reply:
> is this intentional? 
No, forgot to remove the line breaks, fixed via [1712d9b](https://github.com/ballerina-platform/ballerina-distribution/pull/3769/commits/1712d9b7a9915d4d186d080967247e0f4d62097e)